src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1079,7 +1079,7 @@ public void UniqueQueuePerEndpointInstance()
         [ObsoleteEx(
             TreatAsErrorFromVersion = "6",
             RemoveInVersion = "7",
-            ReplacementTypeOrMember = "ScaleOutSettings.InstanceDiscriminator()")]
+            ReplacementTypeOrMember = "EndpointConfiguration.ScaleOut.EndpointInstanceId")]
```

Comment:
should that be `ScaleOut()` and and `InstanceDiscriminator(string discriminator)`?